### PR TITLE
Translate untranslatable string [Delivers #88223448]

### PR DESCRIPTION
**Pull request by andizer**
Finish #304

### CODE REVIEW

File: admin/class-admin-form.php [hunk 1]
```diff
@@ -36,7 +36,11 @@ public static function create_form( $namespace ) {
 	 *
 	 * @return null|string
 	 */
-	public static function end_form( $button_label = 'Save changes', $name = 'submit', $onclick = null ) {
+	public static function end_form( $button_label = null, $name = 'submit', $onclick = null ) {
+		if ( $button_label === null ) {
```
Comment (Petervw) [inline admin/class-admin-form.php]: Isn't it possible to use this instead?
```public static function end_form( $button_label = __( 'Save changes', 'google-analytics-for-wordpress' ), $name = 'submit', $onclick = null ) {```

**Comment by Petervw** [comment]
CR1 done.

File: admin/class-admin-form.php (hunk 1 above)
Comment (andizer) [inline admin/class-admin-form.php]: Nope, you can not evaluate things as parameter.

Comment (Petervw) [inline admin/class-admin-form.php]: Ok, thnx.

**Pull request merged**